code/modules/mob/living/carbon/carbon_update_icons.dm

```diff
@@ -224,76 +224,70 @@
 		var/obj/item/bodypart/BP = X
 		BP.original_owner = WEAKREF(src)
 
-/mob/living/carbon/proc/update_body_parts()
-	//CHECK FOR UPDATE
-	var/oldkey = icon_render_key
-	icon_render_key = generate_icon_render_key()
-	if(oldkey == icon_render_key)
+/mob/living/carbon/proc/update_body_parts(var/update_limb_data)
```

Comment:
auto-doc for this would be appreciated

Reply:
How would you like me to autodoc it?

Reply:
Since you've changed a lot of the implementation, just give a general documentation of that the proc does and when you should call it, and such

"This proc updates the icons of all bodyparts of the mob" etc